react/HistoryRow/styles.styl

```diff
@@ -0,0 +1,40 @@
+@require 'utilities/spaces.styl'
+    
+.HistoryRowCircleWrapper 
+    width 2.6rem
+    height 2.6rem
+    justify-content center
+    display flex
+    align-self start
+
+.HistoryRowCirle
+    border 1px var(--silver) solid
+    background-color #fff
+
+.HistoryRowRevisionLoader
+    display flex
+    justify-content center
+
+.HistoryRowCaption
+    margin-bottom spacing_values.xl
+    padding-left spacing_values.xl
+    padding-right spacing_values.xl
+
+.HistoryRowMedia
+    position relative
+.HistoryRowMedia::before
+    content ''
+    border-left 1px var(--coolGrey) dashed
+    position absolute
+    margin-left 20px
+    //circle is 1rem + 1px border => 18px
+    // 18 + 16px padding => 34
+    top 34px
+    //We have 16px from padding so the next item will be 16px lower
```

Comment:
from which padding ? I don't see a vertical padding in this file ?

I think we should restrain ourselves from commenting when we can have explicit variable names. Comments get easily old and out of date. Moreover variables lead to easier reuse.

```
CIRCLE_HEIGHT=1.125rem // If using this value in .HistoryRowCircle, its box-sizing should be set to border-box so that the border width is accounted in the height
VERTICAL_PADDING_FROM_??=1rem

top: CIRCLE_HEIGHT + VERTICAL_PADDING_FROM_??
```

I think that since the rem are converted implicitly into pixels by multiplying by 16, there will be a bug if the user changes its root font-size (from 16 to another value).

Reply:
You right. Let's see if this one is better ;)